digital-guardbook-ui/src/components/sentry/SentryDisplay.vue

```diff
@@ -56,6 +62,25 @@ function getDateTime(dt: DateTime | string): Date {
         return DateTime.fromISO(dt as string).toJSDate();
     }
 }
+
+async function saveRegistration(): Promise<void> {
+
+    var res = await registerSentry({
+        "sentry": {
+            "id": props.sentry.id,
+            "registration": DateTime.fromISO(props.sentry.registration).toISO()
+        }
+    })
+
+    if (res?.errors == undefined) {
+
+        emit("update:sentry", props.sentry);
+    }
+
+    document.getElementById('registrationModal-close')?.click();
```

Comment:
This feels wrong to me. I have a look, whether there's a better solution.

Reply:
I've fiddled around. It seems to me that there is no quick and easy clean solution for that problem. Therefore, we should keep this as it is and create an new issue that covers this topic. I think we have either to write our own modal component or find an appropriate one.